certbot/util.py

```diff
@@ -23,11 +23,11 @@
 from certbot import lock
 
 try:
-    from collections import OrderedDict
+    from collections import OrderedDict  # type: ignore
 except ImportError:  # pragma: no cover
     # OrderedDict was added in Python 2.7
-    from ordereddict import OrderedDict  # pylint: disable=import-error
-
+    from ordereddict import OrderedDict  as _OrderedDict # pylint: disable=import-error
```

Comment:
I have the same comment as I put in `certbot/plugins/disco.py` about what I think is a slightly cleaner way to handle this.